sw/device/lib/dif/dif_entropy_src.h

```diff
@@ -262,10 +277,9 @@ typedef struct dif_entropy_src_test_stats {
  * @param enabled The enablement state of the entropy source.
  * @return The result of the operation.
  */
-OT_WARN_UNUSED_RESULT
-dif_result_t dif_entropy_src_configure(const dif_entropy_src_t *entropy_src,
-                                       dif_entropy_src_config_t config,
-                                       dif_toggle_t enabled);
+OT_WARN_UNUSED_RESULT dif_result_t dif_entropy_src_configure(
```

Comment:
nit:
```suggestion
OT_WARN_UNUSED_RESULT
dif_result_t dif_entropy_src_configure(
```

Reply:
ah thanks, I will fix this in the follow up PR #13639 that relies on this PR, so as not to re-run CI again on this.